### ISSUE TITLE
Optimize the size of the single-column layout on the desktop

### ISSUE THREAD

**Issue by magicsysrq** [ezra-bible-app/ezra-bible-app#813]
For a comfortable reading experience, I like to have a single column layout to be around 50-55 characters per row. To achieve that in Ezra on the desktop, I need to make the entire window narrow at which point the various controls and buttons get rearranged. Since vertical height is what counts for reading as we progress through a page top to bottom, the interface could do with some improvements to increase vertical space. Please see my attached screenshot.
There is quite a bit of redundant information in this image. The name of the book is shown multiple times. Perhaps the book drop-down could be updated to show the currently selected book instead. Then the tab could show just the name of the translation. For chapter information, the selector should be sufficient. Why include those navigational items in the text itself to steal vertical space?
These are just some suggestions. The main thing is, let's have more vertical space. Have fewer rows of buttons, arrange them on the side, or make them smaller if need be. Alternatively, introduce the ability to have multiple panes within the  main window that can be made narrower. That would also help with comparing multiple translations side by side.
 
<img width="674" alt="ezra" src="https://user-images.githubusercontent.com/67340793/202862031-0ef41552-9215-46b7-bb04-1f0bc00375fd.png">

**Comment by tobias-klein** [comment]
@magicsysrq Thanks for your suggestions!

> For a comfortable reading experience, I like to have a single column layout to be around 50-55 characters per row. To achieve that in Ezra on the desktop, I need to make the entire window narrow at which point the various controls and buttons get rearranged.

Based on some additional styles it should be possible to limit the width of the bible text to a certain maximum width. The question is whether that should be the default then, or rather an option that users can toggle.

> Since vertical height is what counts for reading as we progress through a page top to bottom, the interface could do with some improvements to increase vertical space.

May I ask what kind of screen you are working with? I think this largely depends on the screen size. With a big screen I don't see a big issue with the current interface, with smaller screens I can see your point.

> Please see my attached screenshot. There is quite a bit of redundant information in this image. The name of the book is shown multiple times. Perhaps the book drop-down could be updated to show the currently selected book instead. Then the tab could show just the name of the translation.

I see your point. However, when you switch to another tab I think it is relevant information which exact book / topic is opened in the other tabs. When removed from the tab title, this information would be hidden.

> For chapter information, the selector should be sufficient. Why include those navigational items in the text itself to steal vertical space?

Good point. Generally speaking, the navigational items in the text were introduced for the smartphone use case where you don't use the vertical navigation (due to limited horizontal space in portrait mode). What we could do is only show the chapter navigation in the text if the vertical navigation is not shown. How about that?

> These are just some suggestions. The main thing is, let's have more vertical space. Have fewer rows of buttons, arrange them on the side, or make them smaller if need be. Alternatively, introduce the ability to have multiple panes within the main window that can be made narrower. That would also help with comparing multiple translations side by side.

I definitely see your point regarding the vertical space (depending on screen size).
I'll see what we can do.

There is a separate issue #30 regarding parallel translation display, but I have never gotten around to it yet.

**Comment by magicsysrq** [comment]
> Based on some additional styles it should be possible to limit the width of the bible text to a certain maximum width.
The question is whether that should be the default then, or rather an option that users can toggle.

I definitely like my columns narrow and tall but I realize that may not be the preference for everyone. In printed Bibles the double-column format is still very popular though. Many people dislike single-column Bibles simply because the text is too wide. It's subjective of course so I'm not suggesting that you change the defaults.

> May I ask what kind of screen you are working with? I think this largely depends on the screen size. With a big screen I don't see a big issue with the current interface, with smaller screens I can see your point.

I actually use a 27" 4K screen, so screen real estate is not the problem. I couldn't read a Wikipedia article in a maximized window either. It would be uncomfortably wide.

> I see your point. However, when you switch to another tab I think it is relevant information which exact book / topic is opened in the other tabs. When removed from the tab title, this information would be hidden.

Yes, I realize that now.
 
> Good point. Generally speaking, the navigational items in the text were introduced for the smartphone use case where you don't use the vertical navigation (due to limited horizontal space in portrait mode). What we could do is only show the chapter navigation in the text if the vertical navigation is not shown. How about that?

Yes, that sounds like a good solution.

> There is a separate issue #30 regarding parallel translation display, but I have never gotten around to it yet.

Yes, displaying translations in parallel would be great. I really like Parallel Plus by TheBible.org just because of that feature. The verse-based comparison in Ezra is nice too but being able to compare a full chapter side by side including section headings has its benefits.

Thank you for all your hard work! I haven't used  Ezra for very long but I really like it.

**Comment by tobias-klein** [comment]
I am converting this issue into a discussion, we can create issues for individual and specific enhancements.
